CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.23.0 (2024-11-13)
```

Comment:
Don't we need to make all the metadata changes in this PR?

Reply:
Oh yes! Well remembered!

Reply:
Done!